### PR TITLE
Added section for client IDs

### PR DESCRIPTION
**Pull request by petemoore**
From taskcluster/taskcluster-proxy#19, adding namespace information for client IDs.

### CODE REVIEW

**Comment by djmitche** [comment]
I'll try to merge this into #71.

File: devel/namespaces/index.md [hunk 1]
```diff
@@ -84,6 +84,28 @@ Both are listed here:
...
+logged by taskcluster components.
+
+Until recently, temporary credentials inherited the client ID of the permanent
+credentials that they were created from, but now it is possible to also create
+named temporary credentials with a custom client ID. Both forms of temporary
+credentials are currently in operational use. See [createClient API
+endpoint](http://docs.taskcluster.net/auth/api-docs/#createClient) and
+[Temporary Credentials generation
+process](http://docs.taskcluster.net/auth/temporary-credentials/) for more
+details.
```
Comment (djmitche) [inline devel/namespaces/index.md]: I don't know if an overview of credential types is useful here -- what do you think about the much shorter intro in #71?

**Comment by petemoore** [comment]
Let's go with #71 then - sorry, I should have checked first!

**Comment by petemoore** [comment]
@djmitche Shall we close this now, or did you still want to keep any of it? Fine for me if we ditch it - only if you want it.

**Comment by petemoore** [comment]
@djmitche I'll close this - please reopen if you'd like to keep any of it. Thanks!

**Pull request closed without merging**